### ISSUE TITLE
Different versions of Rn different results

### ISSUE THREAD

**Issue by a22063821** [zakki/Ray#116]
Time and parameters are the same 
## Rn4.14 results
![90001](https://user-images.githubusercontent.com/29286226/29548811-6b65356c-8736-11e7-8ed9-6b8ec3587c0a.png)
# **Top left corner white died**

## Rn4.20.4 results
![9002](https://user-images.githubusercontent.com/29286226/29548895-1f531706-8737-11e7-8ae0-f4297cf4eeaf.png)

# **Top left corner white is alive**

**Why is this situation?
Rn4.14 grasp the superiority
Rn4.20.4 give away the advantage**

**Comment by craytok** [comment]
![00](https://user-images.githubusercontent.com/13972495/29549303-5ad1d7f8-8742-11e7-96d9-f1f7acc7f490.png)

I think there is no problem. 6 black stones at the left side are dead. Rn4.20 made a wise choice

**Comment by craytok** [comment]
![00](https://user-images.githubusercontent.com/13972495/29549426-35aa8e60-8743-11e7-8d52-7082e8230634.png)

I think both of 4.14 and 4.20 made a bad choice.  6 black stones cannot live at your first picture.

![01](https://user-images.githubusercontent.com/13972495/29549470-7e9cd448-8743-11e7-9603-1ce8e127a3df.png)

Just connecting the 4 black stones at the left side is the best choice, i think...

**Comment by crayzok** [comment]
I think Rn4.14 made a bad choice and 4.20 picked not bad choice(stable choice).

Rn4.14 missed fact that 6 black stones cannot live despite of A10 move.

Rn4.20 knew that and she felt that Ko at the left-top corner is quite burdensome...

Because black group at the left-top corner are still not alive too.

And she judged that black still hold a lead this game after black give away 5 black stones at the left side.



![00](https://user-images.githubusercontent.com/13972495/29549794-2d06978e-8745-11e7-9bf8-858bc12fa97e.png)



![01](https://user-images.githubusercontent.com/13972495/29549795-2d1eb04e-8745-11e7-9903-8c00282d2c1f.png)

If I just connect 4 black stones at the left side by putting D9, that's super-Ko.

**Comment by a22063821** [comment]
![908](https://user-images.githubusercontent.com/29286226/29550330-d0aada9a-873f-11e7-95d2-d66887ddbdca.png)

# C-point location will be  better Rn
Rn 10 times choices, 6 times A, 3 times B, 1 times C
**Rn selects C chances low**

**Comment by crayzok** [comment]
yeah, choice of A is stable because it removes Ko but it gives away 4 black stones.

choice of B and C is risky because black group at the left top corner aren't still alive but combative and favorable choice.

B is better than C because of Ko threat(劫材) and B9 move of white.
